test/verify/machineslib.py

```diff
@@ -1188,13 +1188,21 @@ def installWithErrorTest(dialog):
         createTest(TestMachines.VmDialog(self, sourceType='file',
                                          location=config.NOVELL_MOCKUP_ISO_PATH,
                                          memory_size=50, memory_size_unit='MiB',
-                                         storage_size=0, storage_size_unit='MiB',
```

Comment:
This got introduced in the previous commit, it should not be inserted there anyway.